app.py

```
@@ -830,9 +998,199 @@ def check_audio_support():
+    except Exception as e:
+        logger.error(f"Webhook işleme hatası: {str(e)}")
+        return jsonify({
+            'success': False,
+            'error': str(e)
+        }), 500
+
 if __name__ == '__main__':
     try:
-        asgi_app = WsgiToAsgi(app)
+        # Veritabanını hazırla
+        init_db()
```

Ruff (0.8.2)

Undefined name `init_db`

(F821)